server/threadz.go

```diff
@@ -0,0 +1,52 @@
+// Debug tooling. Dumps named profile in response to HTTP request at
```

Comment:
Please rename this file pprof.go.
threadz is only one of the handlers (in addition to "/heapz", "/profilez", etc.)

Reply:
Renamed to http_pprof.go not to confuse with the same but writing to files.